src/repository/exercises-repository.ts

```diff
@@ -1304,1527 +1282,3122 @@ export const db: Schema = {
+      },
+      pastPerfect: {
+        Eu: 'transformei-me',
+        Tu: 'transformaste-te',
+        'Ela/Ele/Você': 'transformou-se',
+        Nós: 'transformámo-nos',
+        'Eles/Elas/Vocēs': 'transformaram-se'
+      }
+    },
+    {
+      english: 'to transport',
+      infinitive: 'transportar',
+      presentSimple: {
+        Eu: 'transporto',
+        Tu: 'transportas',
+        'Ela/Ele/Você': 'transporta',
+        Nós: 'transportamos',
+        'Eles/Elas/Vocēs': 'transportam'
+      },
+      pastPerfect: {
+        Eu: 'transportei',
+        Tu: 'transportaste',
+        'Ela/Ele/Você': 'transportou',
+        Nós: 'transportámos',
+        'Eles/Elas/Vocēs': 'transportaram'
+      }
+    },
+    {
+      english: 'to suffer',
+      infinitive: 'sofrer',
+      presentSimple: {
+        Eu: 'sofro',
+        Tu: 'sofres',
+        'Ela/Ele/Você': 'sofre',
+        Nós: 'sofremos',
+        'Eles/Elas/Vocēs': 'sofrem'
+      },
+      pastPerfect: {
+        Eu: 'sofri',
+        Tu: 'sofreste',
+        'Ela/Ele/Você': 'sofreu',
+        Nós: 'sofremos',
+        'Eles/Elas/Vocēs': 'sofreram'
+      }
+    },
+    {
+      english: 'to hold',
+      infinitive: 'segurar',
+      presentSimple: {
+        Eu: 'seguro',
+        Tu: 'seguras',
+        'Ela/Ele/Você': 'segura',
+        Nós: 'seguramos',
+        'Eles/Elas/Vocēs': 'seguram'
+      },
+      pastPerfect: {
+        Eu: 'segurei',
+        Tu: 'seguraste',
+        'Ela/Ele/Você': 'segurou',
+        Nós: 'segurámos',
+        'Eles/Elas/Vocēs': 'seguraram'
+      }
+    },
+    {
+      english: 'to mean',
+      infinitive: 'significar',
+      presentSimple: {
+        Eu: 'significo',
+        Tu: 'significas',
+        'Ela/Ele/Você': 'significa',
+        Nós: 'significamos',
+        'Eles/Elas/Vocēs': 'significam'
+      },
+      pastPerfect: {
+        Eu: 'signifiquei',
+        Tu: 'significaste',
+        'Ela/Ele/Você': 'significou',
+        Nós: 'significámos',
+        'Eles/Elas/Vocēs': 'significaram'
+      }
+    },
+    {
+      english: 'to save',
+      infinitive: 'salvar',
+      presentSimple: {
+        Eu: 'salvo',
+        Tu: 'salvas',
+        'Ela/Ele/Você': 'salva',
+        Nós: 'salvamos',
+        'Eles/Elas/Vocēs': 'salvam'
+      },
+      pastPerfect: {
+        Eu: 'salvei',
+        Tu: 'salvaste',
+        'Ela/Ele/Você': 'salvou',
+        Nós: 'salvámos',
+        'Eles/Elas/Vocēs': 'salvaram'
+      }
+    },
+    {
+      english: 'to heal',
+      infinitive: 'sarar',
+      presentSimple: {
+        Eu: 'saro',
+        Tu: 'saras',
+        'Ela/Ele/Você': 'sara',
+        Nós: 'saramos',
+        'Eles/Elas/Vocēs': 'saram'
+      },
+      pastPerfect: {
+        Eu: 'sarei',
+        Tu: 'saraste',
+        'Ela/Ele/Você': 'sarou',
+        Nós: 'sarámos',
+        'Eles/Elas/Vocēs': 'sararam'
+      }
+    },
+    {
+      english: 'to provide',
+      infinitive: 'prover',
```

Comment:
use the verb fornecer as translation to 'provide'
and also change the respective example
a empresa forneceu equipamento

Reply:
it exist but very uncommon, never heard it even